cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -500,6 +547,43 @@ public ClusterModel clusterModel(long from,
     return clusterModel;
   }
 
+  /**
+   * Get replica placement information, i.e. each replica resides on which disk of the broker.
+   *
+   * @param clusterModel The cluster model to populate replica placement information.
+   * @param cluster The cluster metadata.
+   * @return A map from topic partition to replica placement information.
```

Comment:
Can we indicate when this is expected to return `null`? -- i.e. we have logic that differs based on this function returning `null` vs `non-null`.

Reply:
This method will not return null,  the null logic is [here](https://github.com/linkedin/cruise-control/blob/8ba6410e24d465b873e2c5f999bb220426654076/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L520)

Reply:
Then do we need this check?: https://github.com/linkedin/cruise-control/blob/8ba6410e24d465b873e2c5f999bb220426654076/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L691

Reply:
yes, we still need this. 
Even through this method does not return null, but this method is [called optionally](https://github.com/linkedin/cruise-control/blob/bdf62c42a2e3f2998215ab2c51d1799e0eee9752/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L524).